opencti-platform/opencti-front/src/components/ExportButtons.jsx

```diff
@@ -182,6 +186,13 @@ class ExportButtons extends Component {
               <FilePdfBox fontSize="small" color="primary" />
             </ToggleButton>
           </Tooltip>
+          {isCustomDashBoard && (
```

Comment:
why not test on workspace property instead of location path ? it can be present only if we are in dashboards view, right ?

Reply:
Right